### ISSUE TITLE
After I put the number the error occured

### ISSUE THREAD

**Issue by Praveensenpai** [Nekmo/telegram-upload#189]
* telegram-upload version: 0.5.1
* Python version: 3.11
* Operating System: Windows
* Dependencies list (run `pip freeze`):

### Description

After I put the number the error occured

### What I Did

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Apps\Python311\Scripts\telegram-upload.exe\__main__.py", line 7, in <module>
  File "C:\Apps\Python311\Lib\site-packages\telegram_upload\exceptions.py", line 64, in wrap
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
  File "C:\Apps\Python311\Lib\site-packages\click\core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Apps\Python311\Lib\site-packages\click\core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "C:\Apps\Python311\Lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Apps\Python311\Lib\site-packages\click\core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Apps\Python311\Lib\site-packages\telegram_upload\management.py", line 139, in upload
    client.start()
  File "C:\Apps\Python311\Lib\site-packages\telegram_upload\client.py", line 110, in start
    return super().start(phone=phone, password=password, bot_token=bot_token, force_sms=force_sms,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Apps\Python311\Lib\site-packages\telethon\client\auth.py", line 128, in start
    else self.loop.run_until_complete(coro)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Apps\Python311\Lib\asyncio\base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Apps\Python311\Lib\site-packages\telethon\client\auth.py", line 184, in _start
    await self.send_code_request(phone, force_sms=force_sms)
  File "C:\Apps\Python311\Lib\site-packages\telethon\client\auth.py", line 433, in send_code_request
    result = await self(functions.auth.SendCodeRequest(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Apps\Python311\Lib\site-packages\telethon\client\users.py", line 30, in __call__
    return await self._call(self._sender, request, ordered=ordered)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Apps\Python311\Lib\site-packages\telethon\client\users.py", line 83, in _call
    result = await future
             ^^^^^^^^^^^^
telethon.errors.rpcerrorlist.FloodWaitError: A wait of 85018 seconds is required (caused by SendCodeRequest)
```

**Comment by Nekmo** [comment]
Hi, you can see the error message at the end. You must wait to authenticate.

```
telethon.errors.rpcerrorlist.FloodWaitError: A wait of 85018 seconds is required (caused by SendCodeRequest)
```

**Comment by Nekmo** [comment]
This is fixed in #111.
